catalystwan/integration_tests/test_find_template_values.py

```diff
@@ -0,0 +1,45 @@
+import os
+import unittest
+from typing import Any, List, cast
+
+from catalystwan.session import create_manager_session
+from catalystwan.utils.feature_template.find_template_values import find_template_values
+
+
+class TestFindTemplateValues(unittest.TestCase):
+    def setUp(self) -> None:
+        self.session = create_manager_session(
+            url=cast(str, os.environ.get("TEST_VMANAGE_URL")),
+            port=cast(int, int(os.environ.get("TEST_VMANAGE_PORT"))),  # type: ignore
+            username=cast(str, os.environ.get("TEST_VMANAGE_USERNAME")),
+            password=cast(str, os.environ.get("TEST_VMANAGE_PASSWORD")),
+        )
+        self.templates = self.session.api.templates._get_feature_templates()
+
+    def test_find_template_value(self):
+        for template in self.templates:
+            definition = template.template_definiton
+            with self.subTest(template_name=template.name):
+                parsed_values = find_template_values(definition)
```

Comment:
Address the underlying issues causing the test function to fail for our demo vManage.

Reply:
Please consider to write this as unit test to not rely on having fully configured vManage